Content.Server/Storage/Components/ServerStorageComponent.cs

```diff
@@ -628,15 +629,11 @@ void IExAct.OnExplosion(ExplosionEventArgs eventArgs)
             }
         }
 
-        protected void PlaySoundCollection(string? name)
+        private void PlaySoundCollection()
         {
-            if (string.IsNullOrEmpty(name))
-            {
-                return;
-            }
-
-            var file = AudioHelpers.GetRandomFileFromSoundCollection(name);
-            SoundSystem.Play(Filter.Pvs(Owner), file, Owner, AudioParams.Default);
+            // TODO this doesn't compile or work
+            if(StorageSoundCollection?.TryGetSound(out var sound))
+                SoundSystem.Play(Filter.Pvs(Owner), sound, Owner, AudioParams.Default);
```

Comment:
this is the current issue:
StorageSoundCollection?.TryGetSound(out var sound) doesn't work because it is bool?
if I try 
StorageSoundCollection?.TryGetSound(out var sound) == true
then it complains about sound being unassigned

Reply:
solved, simply used GetSound everywhere